http-client-core/src/main/java/io/micronaut/http/client/interceptor/HttpClientIntroductionAdvice.java

```diff
@@ -332,6 +337,14 @@ private Object handlePublisher(MethodInvocationContext<Object, Object> context,
         } else {
             publisher = httpClientResponsePublisher(httpClient, requestPublisher, returnType, errorType, valueType);
         }
+
+        publisher = Flux.from(publisher)
```

Comment:
make this entire statement conditional on isDebugEnabled

Reply:
done